app/src/main/java/eu/kanade/tachiyomi/data/source/online/OnlineSource.kt

```diff
@@ -54,6 +54,11 @@ abstract class OnlineSource(context: Context) : Source {
     abstract val lang: Language
 
     /**
+     * If the Source has Support for Latest Updates
+     */
+    abstract val supportsLatest : Boolean
```

Comment:
Initially I made this change thinking it would have less overhead but I later realized that come compile time its just a get() anyway so its the same